src/Sulu/Bundle/WebsiteBundle/Twig/Core/UtilTwigExtension.php

```diff
@@ -26,14 +27,22 @@ public function getFilters()
         return [
             new TwigFilter('sulu_util_multisort', 'Sulu\Component\Util\SortUtils::multisort'),
             new TwigFilter('sulu_util_filter', 'Sulu\Component\Util\ArrayUtils::filter'),
-            new TwigFilter('sulu_util_domain_info', [$this, 'extract']),
+            new TwigFilter(
+                'sulu_util_domain_info',
+                [$this, 'extract'],
+                ['deprecation_info' => new DeprecatedCallableInfo(package: 'sulu/sulu', version: 2.3)],
```

Comment:
not supported by our range of supported twig versions

Reply:
Then just deprecated => true?

Reply:
if that works for all our supported twig versions

Reply:
Yes, that behaviour works in the old versions and (deprecated) in the new versions.